runtime/common/src/lib.rs

```diff
@@ -795,3 +809,13 @@ pub mod rewards {
 		pub const SingleCurrencyMovement: u32 = 1;
 	}
 }
+
+/// Converts the given number to percent
+pub const fn to_percent(x: u128) -> sp_arithmetic::FixedI64 {
```

Comment:
Super NIT. Maybe it fits better in cfg_utils/maths or similar (?)

(also applies to `to_ppm()`)

Reply:
Absolutely agree: [c7be7f9](https://github.com/centrifuge/centrifuge-chain/pull/1828/commits/c7be7f968abfb789ddb949affc9a6cce98a7ed81)

Reply:
Thanks for tackle my Nit thing 🙌🏻